types/api/v1beta1/tracee.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+option go_package = "github.co/aquasecurity/tracee/api/v1";
+
+package tracee.v1beta1;
+
+message GetVersionRequest {
+}
+
+message GetVersionResponse {
+    string version = 1;
+}
+
+service TraceeService {
+    rpc GetVersion(GetVersionRequest) returns (GetVersionResponse);
+}
```

Comment:
nit: new line

```suggestion
}

```

Reply:
Is there a default agreed for this we follow?

Reply:
I think we're following the github alert.

![image](https://github.com/aquasecurity/tracee/assets/3372117/62b171aa-1511-4f38-8100-0a13f4e888ea)

But indeed not discussed between us.

Reply:
🤔 I don't think it is an alert as something bad hahaha, but I don't mind adding a new line, maybe we can find some protobuf linter that keeps an eye on this.